src/calculateRentalCost.js

```diff
@@ -4,7 +4,23 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const BASE_PRICE = 40;
+  const NUMBER_OF_DAYS = days;
+  const AMOUNT = NUMBER_OF_DAYS * BASE_PRICE;
+  const LONG_TERM = 7;
+  const LONG_TERM_DISCOUNT = 50;
+  const SHORT_TERM = 3;
+  const SHORT_TERM_DISCOUNT = 20;
+
+  if (NUMBER_OF_DAYS >= LONG_TERM) {
+    return AMOUNT - LONG_TERM_DISCOUNT;
+  }
+
+  if (NUMBER_OF_DAYS >= SHORT_TERM) {
+    return AMOUNT - SHORT_TERM_DISCOUNT;
```

Comment:
There is a logical issue here. If the number of days is greater than or equal to 7, the long-term discount is applied, and the function returns immediately. This means the short-term discount will never be applied for rentals longer than 7 days. Consider revising the logic to ensure that the correct discount is applied based on the number of days.